collaboration-guide.md

```diff
@@ -0,0 +1,30 @@
+## Project organization
+
+As you can see in the ./README.md, our project is organized with directories. Each one has specific files related to the develpment. Here in this file we will present the project workflow. It is based on [GitHub Flow](https://guides.github.com/introduction/flow/) (read this up), with some additional steps added to maintain rastreability of activities.
+
+## Workflow
+
+* An issue is an project activity that needs to be done. One issue is assigned to one or more collaborators.
+* Each issue has a branch, that contains changes made to finish the activity. So, once you are assigned to an issue and are going to work on it, create a branch that will contain the changes.
+* Once you make all necessary changes, create a Pull Request, where the team will discuss about your changes, and, if necessary, propose changes. In this state, you can continue to push changes to your branch in light of discussion and feedback about your commits.
+* After the PR is approved by reviewers, it is ready to be merged to `master`. Once it is, the issue is concluded.
```

Comment:
The merge type should be Squash and Merge

Reply:
Done